jsx/MultiSelectDropdown.js

```diff
@@ -5,11 +5,24 @@
  For generic SelectDropdown, see Select in Form.js
  */
 
-let SelectField = React.createClass({
-  toggleCheckbox: function() {
+import React, {Component} from 'react';
+
+class SelectField extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+
+    };
```

Comment:
I don't think we need a state declaration if this is a stateless component. Same thing for all the ones below.